src/removeTask.js

```diff
@@ -0,0 +1,13 @@
+import todoArray from './__mocks__/todoarray';
+
+const removeTask = (list) => {
+  const index = Number(list.id);
+  list.remove();
+  todoArray.splice(index, 1);
+  todoArray.forEach((element, index) => {
+    element.index = index;
+  });
+  // localStorage.setItem('data', JSON.stringify(todoArray));
```

Comment:
- [ ] @team kindly remove the commented code, it's not good practice to have commented code in your codebase.